INSTALL

```diff
@@ -609,6 +611,7 @@ The specific libraries and their options are:
 
 - PUMI (optional), used when MFEM_USE_PUMI = YES.
   URL: https://scorec.rpi.edu/pumi
+       https://github.com/SCOREC/core
   Options: PUMI_OPT, PUMI_LIB.
   Versions: PUMI >= 2.2.0.
```

Comment:
I assume this version is no longer sufficient. Is there a released PUMI version that has the changes required for this PR?

Reply:
The latest version of PUMI, as far as I know (@cwsmith  can comment on this), is 2.2.2 and that does not include these changes. I think the preferred way of using PUMI is to have the latest updates by pulling from either `master` or `develop` branch. I added a note in `INSTALL` here 
https://github.com/mortezah/mfem/blob/40fb14767668b793de74b5f4b12025da26f12e2c/INSTALL#L429 to that effect

Reply:
This is not sufficient.

Please make sure the text after versions is correct, so users can indeed reproduce the build by themselves.

Maybe something like 

```
  Versions: PUMI >= 2.3.0
```

or

```
  Versions: Any version after commit xxxxxxx in PUMI's develop branch
```

Reply:
I have updated `INSTALL` in the following commit

https://github.com/mortezah/mfem/commit/8d458616a77bc812c7d7626c519f9522ee9e3d38

Reply:
PUMI v2.2.3 now exists:
https://github.com/SCOREC/core/tree/v2.2.3
https://github.com/SCOREC/core/issues/308

Reply:
In that case my preference will be to say `PUMI >= 2.3.0`

Reply:
I will update that shortly.

Reply:
I will update that shortly.

Reply:
I pushed the change here https://github.com/mortezah/mfem/commit/a9bcd488cb99c88f76877e3840c1557e4ea1ad45